lib/fortnox/api/models/article.rb

```diff
@@ -0,0 +1,147 @@
+        attribute :eu_vat_account, Types::Sized::Integer[ 0, 99_99 ]
+
+        #ExportAccount Account number for the sales account outside EU
+        # The number must be of an existing account.
+        attribute :export_account, Types::Sized::Integer[ 0, 99_99 ]
+
+        #Height The height of the article in millimeters
+        attribute :height, Types::Sized::Integer[ 0, 99_999_999 ]
+
+        #Housework If the article is housework
+        attribute :housework, Fortnox::API::Types::Nullable::Boolean
+
+        #HouseWorkType The type of house work.
+        attribute :house_work_type, Types::HouseWorkType
+
+        #Manufacturer The manufacturer of the article
+        attribute :manufacturer, Fortnox::API::Types::Sized::String[ 50 ]
+
+        #ManufacturerArticleNumber The manufacturer's article number
+        attribute :manufacturer_article_number, Fortnox::API::Types::Sized::String[ 50 ]
+
+        #Note Text note
+        attribute :note, Fortnox::API::Types::Sized::String[ 10_000 ]
+
+        #PurchaseAccount Account number for purchase.
+        # The number must be of an existing account.
+        attribute :purchase_account, Types::Sized::Integer[ 0, 99_99 ]
+
+        #PurchasePrice Purchase price of the article
+        attribute :purchase_price, Fortnox::API::Types::Sized::Float[ 0.0, 99_999_999_999_999.9 ]
+
+        #QuantityInStock Quantity in stock of the article
+        attribute :quantity_in_stock, Fortnox::API::Types::Sized::Float[ 0.0, 99_999_999_999_999.9 ]
+
+        #ReservedQuantity Reserved quantity of the article
+        attribute :reserved_quantity, Fortnox::API::Types::Nullable::Float.with( read_only: true )
+
+        #SalesAccount Account number for the sales account in Sweden.
+        # The number must be of an existing account.
+        attribute :sales_account, Types::Sized::Integer[ 0, 99_99 ]
+
+        #SalesPrice Price of article for its default price list
+        attribute :sales_price, Fortnox::API::Types::Nullable::Float.with( read_only: true )
+
+        #StockGoods If the article is stock goods
+        attribute :stock_goods, Fortnox::API::Types::Nullable::Boolean
+
+        #StockPlace Storage place for the article
+        attribute :stock_place, Fortnox::API::Types::Sized::String[ 100 ]
+
+        #StockValue Value in stock of the article
+        attribute :stock_value, Fortnox::API::Types::Nullable::Float.with( read_only: true )
+
+        #StockWarning When to start warning for low quantity in stock
+        attribute :stock_warning, Fortnox::API::Types::Sized::Float[ 0.0, 99_999_999_999_999.9 ]
+
+        #SupplierName Name of the supplier
+        attribute :supplier_name, Fortnox::API::Types::Nullable::String.with( read_only: true )
+
+        #SupplierNumber Supplier number for the article.
+        # The number must be of an existing supplier.
+        attribute :supplier_number, Fortnox::API::Types::Nullable::String.with( read_only: true )
```

Comment:
Should not be read only according to the documentation.

Reply:
Too much copy-paste 😁